service/src/main/java/bio/terra/pipelines/stairway/imputation/RunImputationJobFlightMapKeys.java

```diff
@@ -14,6 +14,12 @@ public abstract class RunImputationJobFlightMapKeys {
   public static final String WDL_METHOD_VERSION = "wdl_method_version";
   public static final String PIPELINE_RUN_OUTPUTS = "pipeline_run_outputs";
 
+  // keys to determine which Stairway hooks to run
+  public static final String DO_SET_PIPELINE_RUN_STATUS_FAILED_HOOK =
```

Comment:
i would consider moving these flight wide keys to their own class.  Also do you know why these keys are `final String`s and the other key class is an enum?

Reply:
yea i dont really have a preference as to which id just rather be consistent if we can. I just noticed that we have two ways of doing it and didnt know if there was a good reason for it